src/middlewared/middlewared/plugins/acme_protocol.py

```diff
@@ -414,6 +418,8 @@ def update_txt_record(self, data):
 
     @private
     def update_txt_record_route53(self, domain, challenge, key, access_key_id, secret_access_key):
+        self.middleware.call_sync('system.advanced.will_perform_network_activity', 'acme')
```

Comment:
We should do this above at line 399 as this will be specific to route53, however middleware consumes the above endpoint and these are called internally by the service